docs/book/src/getting_started/README.md

```diff
@@ -2,19 +2,35 @@
 
 There are two basic paths to getting started with Leptos:
 
-1. Client-side rendering with [Trunk](https://trunkrs.dev/)
-2. Full-stack rendering with [`cargo-leptos`](https://github.com/leptos-rs/cargo-leptos)
+1. Client-side rendering with [Trunk](https://trunkrs.dev/) - a great option if you just want to make a snappy website with Leptos, or work with a pre-existing server or 3rd-party API's.
+2. Full-stack / Universal server-side rendering with [`cargo-leptos`](https://github.com/leptos-rs/cargo-leptos) - a great option for building a CRUD-style website or a web-app with Rust powering both frontend and backend. Universal apps rely on either Actix-web or Axum on the server (so you may want to keep their docs handy, too), or you can even go Universal WASM/WASI with the Leptos and Axum integration (more on these options later).
 
-For the early examples, it will be easiest to begin with Trunk. We’ll introduce
-`cargo-leptos` a little later in this series.
+In Part 1 of this book, we'll start with Client-side rendering  Leptos sites and building reactive UI's using `Trunk` to serve our JS+Wasm bundle to the browser.
 
-If you don’t already have it installed, you can install Trunk by running
+We’ll introduce `cargo-leptos` in Part 2 of this book, which is all about working with the full power of Leptos in its Full-stack, SSR mode.
+
+```admonish note
+If you're coming from the Javascript world and terms like Client-Side Rendering (CSR) and server-side rendering (SSR) are unfamiliar to you, the easiest way to understand the difference is by analogy:
+
+CSR is similar to working with React (or a 'signals'-based framework like SolidJS), and focuses on producing a client-side UI which you can use with any tech stack on the server.
+
+SSR is like working with a full-stack framework like Next.js in the React world (or Solid's "SolidStart" framework) - SSR helps you build sites and apps using JS on both the client and the server.
```

Comment:
Maybe change from `SSR is like working with...` to `SSR is similar to working with...` to avoid using "like" twice in a row and to parallel the previous CSR item.

Also it might be confusing to say that "SSR helps you build sites and apps using JS on both the client and the server" when we aren't really concerned with using JS on either the client or the server in Leptos. I understand this section is directed toward JS devs, but maybe this could be reworded to be more language-agnostic -- something like:
"SSR lets you build apps which are rendered on the server and sent to the client, potentially making client/server communication more streamlined." I'm not sure, something like that.

Reply:
Looks great to me! :+1: